.github/workflows/cloudbeat-ci.yml

```diff
@@ -32,7 +32,11 @@ jobs:
           go-version-file: .go-version
 
       - name: Init Hermit
-        run: ./bin/hermit env -r >> $GITHUB_ENV
+        run: |
+          # Pre-fetch hermit binaries otherwise pre-commit can fail when running multiple commands in parallel. This
+          # leads to hermit attempting to download the same binary multiple times which results in a race condition.
```

Comment:
Super optional, but should we put the comments as yaml comments before run and not as shell comments inside of it?